packages/broker/src/config/ConfigWizard.ts

```diff
@@ -1,254 +1,591 @@
+            .replace(/[^\da-z]/gi, '')
+
+        let config: ConfigFile = {
+            $schema: formSchemaUrl(CURRENT_CONFIGURATION_VERSION),
+            client: {
+                auth: {
+                    privateKey,
+                },
+            },
+            plugins: {
+                ...operatorPlugins,
+                ...pubsubPlugins,
+            },
+            httpServer,
+            apiAuthentication: {
+                keys: [apiKey]
+            }
+        }
+
+        if (network === 'mumbai') {
+            config = getMumbaiConfig(config)
+        }
+
+        persistConfig(
+            storagePath,
+            config,
+        )
+
+        log(`
+            >
+            > ~ *Congratulations, you've setup your Streamr node!*
+            > Your node address is ${chalk.greenBright(nodeAddress)}
+            > Your node's generated name is ${chalk.greenBright(getNodeMnemonic(privateKey))}
+        `)
+
+        if (operator) {
+            const resume = progress((f) =>
+                style(`> Your node address has *${f} MATIC* _– checking balance…_`)
+            )
+
+            try {
+                const balance = await getNativeBalance(network, nodeAddress)
+
+                const content = `Your node address has *${utils
+                    .formatEther(balance)
+                    .replace(/\.(\d+)/, (f) => f.substring(0, 3))} MATIC*`
+
+                resume()
+
+                if (balance.lt(MIN_BALANCE)) {
+                    log(`
+                        > ! ${content}. You'll need to fund it with a small amount of MATIC tokens.
+                    `)
+                } else {
+                    log(`> ${content}`)
+                }
+            } catch (e) {
+                resume()
+
+                log('> x Failed to fetch node\'s balance')
+            }
+        }
+
+        if (operator) {
+            log('> ')
+
+            const resume = progress((f) =>
+                style(`
+                    > _Checking if your node has been paired with your Operator… ${f}_
+                `)
+            )
+
+            try {
+                const nodes = await getOperatorNodeAddresses(network, operator)
+
+                resume()
+
+                const hub =
+                    network === 'polygon'
+                        ? 'https://streamr.network/hub'
+                        : 'https://mumbai.streamr.network/hub'
+
+                if (nodes) {
+                    if (!nodes.includes(nodeAddress.toLowerCase())) {
+                        log(
+                            '> ! You will need to pair your node with your Operator:'
+                        )
+                    } else {
+                        log('> Your node has been paired with your Operator:')
+                    }
+                } else {
+                    log(`
+                        > x Your Operator could not be found on the **${network.replace(/\w/, (l) => l.toUpperCase())}** network, see
```

Comment:
What the `network.replace(/\w/, (l) => l.toUpperCase())` does? E.g. does it capitalize the string (if that is what it does, could use https://lodash.com/docs/4.17.15#capitalize)